packages/PyTrilinos/CMakeLists.txt

```diff
@@ -144,14 +144,14 @@ IF(PyTrilinos_ENABLE_NOX)
   APPEND_SET(PyTrilinos_MODULES  NOX.Abstract  )
   APPEND_SET(PyTrilinos_MODULES  NOX.StatusTest)
   APPEND_SET(PyTrilinos_MODULES  NOX.Solver    )
-  IF(NOX_ENABLE_Epetra)
+  IF(NOX_ENABLE_ABSTRACT_IMPLEMENTATION_EPETRA)
```

Comment:
This CMake option should be defined, since NOX is enabled.  However, you might still want to put an `ASSERT_DEFINED` above this, like so:
```
ASSERT_DEFINED(NOX_ENABLE_ABSTRACT_IMPLEMENTATION_EPETRA)
IF(NOX_ENABLE_ABSTRACT_IMPLEMENTATION_EPETRA)
```
That way, if NOX ever removes or renames that option, you would get a clear error message.

Reply:
This particular code block is protected by

    IF(PyTrilinos_ENABLE_NOX)

at line 141. So `cmake` should not see it if NOX is not enabled.